subsys/net/lib/conn_mgr/conn_mgr.c

```diff
@@ -191,6 +209,127 @@ K_THREAD_DEFINE(conn_mgr, CONFIG_NET_CONNECTION_MANAGER_STACK_SIZE,
 		(k_thread_entry_t)conn_mgr_handler, NULL, NULL, NULL,
 		THREAD_PRIORITY, 0, 0);
 
+static void conn_mgr_all_if_up_cb(struct net_if *iface, void *user_data)
```

Comment:
The function names are quite long, would it harm if we make them shorter. So something like
`conn_mgr_up_cb()` etc. The "all_if" seems redundant.